inspirehep/records/utils.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""INSPIRE module that adds more fun to the platform."""
+from inspire_utils.name import ParsedName
+from inspire_utils.record import get_value, get_values_for_schema
+
+
+def get_pid_from_record_uri(record_uri):
+    """Transform a URI to a record into a (pid_type, pid_value) pair."""
+    parts = [part for part in record_uri.split("/") if part]
+    try:
+        pid_type = parts[-2][:3]
+        pid_value = parts[-1]
+    except IndexError:
+        return None
+
+    return pid_type, pid_value
+
+
+def get_author_with_record_facet_author_name(author):
```

Comment:
aren't these dups https://github.com/inspirehep/inspirehep/blob/master/inspirehep/records/marshmallow/authors/common/facet_author_name.py

Reply:
Fixed.

Reply:
hmm how is this fixed? :/

Reply:
It was fixed in opposite way :P now it is fixed. 
`get_author_with_record_facet_author_name` form utils removed. Using staticmethod from `AuthorsMetadataSchemaV1`. 
Also same thing with `get_author_display_name`.